hr_employee_calendar_planning/migrations/13.0.1.2.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE resource_calendar
```

Comment:
You have to take into account that you can come from `12.0.1.1.0` and this was already done. One possible technique is to check `version` that stores version from which you come, so if it's `>= 13.0.1.0.0`, then you can perform this.